src/main/resources/application.yml

```diff
@@ -165,9 +160,17 @@ feature_toggles:
   async_event_operations_enabled: false
   directions_questionnaire_enabled: false
 
+logging.level.uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi: DEBUG
```

Comment:
Do we want this change to prod?

Reply:
added to trace Feign Client calls during debugging. Will not be merged to Master